src/solvers/dgsem_tree/subcell_limiters.jl

```diff
@@ -108,4 +152,13 @@ function get_node_variables!(node_variables, limiter::SubcellLimiterIDP,
 
     return nothing
 end
+
+@inline function get_variable_index(variable, variable_names)
+    for (i_, variable_) in enumerate(variable_names)
+        if variable == variable_
+            return i_
+        end
+    end
+    error("$variable is no valid variable.")
+end
```

Comment:
Maybe move this to `src/equations/equations.jl`?

Reply:
And also add a small explanatory comment?

Reply:
Thanks. I additionally rewrote the function like
```
@inline function get_variable_index(variable, equations, conversion_function = cons2cons)
    index = findfirst(==(variable), varnames(conversion_function, equations))
    if isnothing(index)
        error("$variable is no valid variable.")
    end

    return index
end
```